### ISSUE TITLE
Added Child = 4 Entities

### ISSUE THREAD

**Issue by foxalerts23** [jcgoette/baby_buddy_homeassistant#88]
Integration was working previously, after adding a child to Baby Buddy there are only 4 entities and not assigned to a child.
I have deleted the integration and re-added, but that didn't do anything.

I enabled debug logging, but the only log is the following:

Unexpected error fetching babybuddy data: 'count'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 225, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 181, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/babybuddy/__init__.py", line 198, in async_update
    if children_list[ATTR_COUNT] < len(self.child_ids):
KeyError: 'count'

**Comment by jcgoette** [comment]
There are no other log messages?

**Comment by foxalerts23** [comment]
These are the only logs.  When I search for the word "baby" as well as when I reload the integration this pops to the top of the logs with nothing else.

**Comment by jcgoette** [comment]
Added just a little more debugging in c046c7ade1959cacbf78269aa8fef0676633dce3 if you want to update and post more logs.

**Comment by foxalerts23** [comment]
No additional logs, looks to be the exact same, but one of the lines incremented.

Unexpected error fetching babybuddy data: 'count'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 225, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 181, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/babybuddy/__init__.py", line 199, in async_update
    if children_list[ATTR_COUNT] < len(self.child_ids):
KeyError: 'count'

**Comment by jcgoette** [comment]
Can you post how you turned on debug logging for BB?

**Comment by foxalerts23** [comment]
Settings > Devices & Services > click on the vertical ellipses for Baby Buddy and enabled debug logging.  I then reloaded the integration. I ended up restarting home assistant after updating an integration and checked that the debug icon was still displayed on the Baby Buddy integration, checked logs again and then I posted.

**Comment by jcgoette** [comment]
Can you upload the log then?

https://www.home-assistant.io/docs/configuration/troubleshooting/#debug-logs-and-diagnostics

**Comment by foxalerts23** [comment]
Lots of other logs in here, but at least you have all the info.
[home-assistant_babybuddy_2023-01-09T04-30-02.490Z.log](https://github.com/jcgoette/baby_buddy_homeassistant/files/10370379/home-assistant_babybuddy_2023-01-09T04-30-02.490Z.log)

**Comment by foxalerts23** [comment]
I added that to my configuration file, disabled the integration, deleted the integration, and rebooted.
I then re-added the integration.
The integration resulted in the same 4 entities.
It shows that debugging is on, if I disable it, I download a 36MB log file, with TONS of data, too much for me to feel comfortable uploading without sanitizing.  I will work on sanitizing.

**Comment by jcgoette** [comment]
I don't need the full log. Just anything BB related.

**Comment by foxalerts23** [comment]
Sorry for the late reply, but I was able to scrub this tonight (I see you just happened to close it too, although I am seeing that now as I'm posting). 
[home-assistant_babybuddy_2023-01-11-scrub.log](https://github.com/jcgoette/baby_buddy_homeassistant/files/10473671/home-assistant_babybuddy_2023-01-11-scrub.log)

I had to delete a lot to make this usable and hopefully I didn't remove too much, but there is something interesting happening in line 34.  To me it looks like forgetting the port in the URL.  Because I am using a Synology as a docker host, which defaults to using ports 5000 and 5001, it looks like these ports are showing up.
